R/PipeOpFFS.R

```diff
@@ -54,9 +54,9 @@ PipeOpFFS = R6Class("PipeOpFFS",
     #'   List of hyperparameter settings, overwriting the hyperparameter settings that would
     initialize = function(id = "ffs", param_vals = list()) {
       param_set = ps(
-        drop = p_lgl(default = FALSE, tags = c("train", "predict")),
-        left = p_dbl(default = -Inf, tags = c("train", "predict")),
-        right = p_dbl(default = Inf, tags = c("train", "predict")),
+        drop = p_lgl(tags = c("train", "predict", "required")),
+        left = p_dbl(tags = c("train", "predict", "required")),
+        right = p_dbl(tags = c("train", "predict", "required")),
```

Comment:
Here we now need to: 
1. Set the initial values accordingly (`param_set$set_values(drop = FALSE, left = -Inf, right = Inf)`. And change the documentation of the parameters (from "default" to "initial value")